source/DG.Core/Extensions/ApplicationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace DG.Core.Extensions
+{
+    public static class ApplicationExtensions
+    {
+        public static string ConstructUniqueId(string application, string instanceName)
+        {
+            return $"{application}/{instanceName}";
```

Comment:
We need to think about instance id, maybe we should use "-" or "^" or "|" instead of "/" in order to escape problems in query. Or some other variants.

Reply:
Included test to check it, please see test ShouldWorkWithApplicationsWithSpecializedNames